### PR TITLE
CLDR-16661 Use /util/DowngradePaths.java when importing votes

### PR DESCRIPTION
**Pull request by btangmu**
-Filter by DowngradePaths.lookingAt when importing old winning or losing votes

-Revise DowngradePaths to use getUTF8Data(downgrade.txt); move downgrade.txt to util/data

-Make sure vote table exists before importing old votes, call STFactory.setupDB

-Make STFactory.setupDB public (there should be a better way, required for table access)

CLDR-16661

- [x] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by macchiati — COMMENTED** [review]
Looks good to me, but Steven should review the ST stuff.

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java [hunk 1]
```diff
@@ -643,6 +643,7 @@ private void processRequest(
                         send(r, out);
                     } else if (what.equals(WHAT_OLDVOTES)) {
                         mySession.userDidAction();
+                        CookieSession.sm.getSTFactory().setupDB();
```
Comment (srl295) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java:646]: Probably ok for now

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java [hunk 2]
```diff
@@ -1992,7 +2009,7 @@ private void doAutoImportOldWinningVotes(SurveyJSONWrapper r, User user, SurveyM
             return;
         }
         alreadyAutoImportedVotes(user.id, "set");
-
+        CookieSession.sm.getSTFactory().setupDB();
```
Comment (srl295) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java:2012]: For the record could just skip if not existing. But ok.

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/DowngradePaths.java [hunk 3]
```diff
@@ -1,15 +1,9 @@
...
-import org.unicode.cldr.draft.FileUtilities;
-
 public class DowngradePaths {
 
     static LocalePathValueListMatcher data =
-            LocalePathValueListMatcher.load(
-                    Paths.get(
-                            FileUtilities.getRelativeFileName(
-                                    DowngradePaths.class, "downgrade.txt")));
+            LocalePathValueListMatcher.load(CldrUtility.getUTF8Data("downgrade.txt").lines());
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/util/DowngradePaths.java:6]: +1 this is what I expected here

**Review by srl295 — APPROVED** [review]

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java (hunk 2 above)
Comment (btangmu) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java:2012]: I don't think "could just skip": if the new table doesn't exist yet, the old votes won't get imported unless the table gets created here.

**Pull request merged**